CHANGELOG.md

```diff
@@ -31,7 +31,11 @@ A brief description of the categories of changes:
 
 {#v0-0-0-fixed}
 ### Fixed
-- Nothing yet
+- The extension evaluation has been adjusted to always generate the same lock
+  file irrespective if `experimental_index_url` is set by any module or not.
+  Fixes [#2268](https://github.com/bazelbuild/rules_python/issues/2268). A known
+  issue is that it may break `bazel query` and in these usecases it is advisable
+  to use `cquery` until we have `sdist` cross-building from source fully working.
```

Comment:
download_only=True is also an option, right? Assuming one is OK with not having sdists

Reply:
It could be, but I haven't tested with regular requirements files, let me add a unit test to see how that would get wired through.